nifi-nar-bundles/nifi-extension-utils/nifi-database-utils/src/main/java/org/apache/nifi/util/db/JdbcCommon.java

```diff
@@ -345,8 +345,11 @@ public static long convertToAvroStream(final ResultSet rs, final OutputStream ou
                         byte[] bytes = rs.getBytes(i);
                         ByteBuffer bb = ByteBuffer.wrap(bytes);
                         rec.put(i - 1, bb);
-
-                    } else if (value instanceof Byte) {
+					} else if (javaSqlType == 100) { // Handle Oracle BINARY_FLOAT data type
```

Comment:
The changes have tab characters rather than spaces, which causes a CheckStyle violation. I will convert them before merging